.github/workflows/linux.yml

```diff
@@ -0,0 +1,25 @@
+name: Linux desktop CI
+
+on:
+  pull_request: 
+    types: [synchronize, review_requested, ready_for_review]
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-building:
+    if: github.event.pull_request.draft == false
+    runs-on: [ self-hosted, desktop-kdrive-tests ]
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4.1.1
+        with:
+          submodules: recursive
+
+      - name: Grant execute permission for building script
+        run : chmod +x ./infomaniak-build-tools/linux/build-ci.sh
+      - name: Build App
```

Comment:
We may uniformise names: either each word in the name starts with an upper case, or only the first word does.